index.js

```diff
@@ -244,9 +286,15 @@ function getGermanCars(/* code here */) {
  *   return num * 2
  * }
 */
-const sum = null; // code here!
-const addFive = null; // code here!
-const argTimesTwo = null; // code here!
+const sum = ((a,b) => 
+  {return a + b;}
+  ); // code here!
+const addFive = ((num) =>
+{return num + 5}
+); // code here!
+const argTimesTwo = ((num) =>
+{return num * 2}
+); // code here!
```

Comment:
As discussed in our 1:1 review session, you can leave out the curly braces and the return statements on the one-line arrow functions